project/Dependencies.scala

```diff
@@ -245,7 +245,7 @@ object Dependencies {
         Compile.slf4jApi,
         TestDependencies.scalatest.value)
 
-  val remoteDependencies = Seq(netty, aeronDriver, aeronClient)
+  val remoteDependencies = Seq(aeronDriver, aeronClient)
```

Comment:
Doesn't the netty entry completely go away above as well?

Reply:
still used in multi-jvm testkit